rush.json

```diff
@@ -7,7 +7,7 @@
     "strictPeerDependencies": false
   },
 
-  "nodeSupportedVersionRange": ">=8.9.4 <9.0.0 || >=10.9.0 <11.0.0",
+  "nodeSupportedVersionRange": ">=8.9.4 <9.0.0 || >=10.9.0 <12.0.0",
```

Comment:
Is there a reason we want Node 11?

Reply:
I don't want to limit the version of node here. It's actually irrelevant since nothing I'm doing here should limit us from upgrading to node 11

Reply:
In that case you should probably just go with >=8.9.4?